src/Abstractions/NexusMods.Abstractions.Loadouts/LoadoutFile.cs

```diff
@@ -0,0 +1,25 @@
+using JetBrains.Annotations;
+using NexusMods.Abstractions.MnemonicDB.Attributes;
+using NexusMods.MnemonicDB.Abstractions.Models;
+
+namespace NexusMods.Abstractions.Loadouts;
+
+/// <summary>
+/// Represents a file in a Loadout.
+/// </summary>
+[Include<LoadoutItemWithTargetPath>]
+[PublicAPI]
+public partial class LoadoutFile : IModelDefinition
```

Comment:
Should we add a classification to this to distinguish it from DeletedFile and later GeneratedFile?
Something like NormalFile or ClassicFile?
This all doesn't sound great to me, but there is still an ambiguity problem with `LoadoutFile` with respect to the other "file" types.